packages/compass-crud/package.json

```diff
@@ -108,6 +108,7 @@
     "mongodb-ace-autocompleter": "^0.4.12",
     "mongodb-ace-mode": "^0.4.0",
     "mongodb-ace-theme-query": "0.0.2",
+    "mongodb-client-encryption": "^1.2.3",
```

Comment:
This is a bit surprising – the driver should not require this to be present unless we’re actually doing anything with FLE … are we?

Reply:
Hmm, yeah, no, I don't think so... Let me double-check that I actually got the reason for this right in the first place

Reply:
Yeah, so seems it's a breaking* change introduced in driver here https://github.com/mongodb/node-mongodb-native/pull/2760/files#diff-c088451b2b6c514ba0f163c7c00dfae50dc86c534feeffe18cf618749af66da0R15

\* - breaking only for webpack because it tries to bundle mongodb-client-encryption and because this time it's a top level import it throws instead of warning as it does with v3.6.3